.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Thank you for your contribution to OpenRA!
+
+Please be aware that we do not have enough project maintainers to match the rate of contributions, so it may take several days before somebody is able to respond to your Pull Request.
```

Comment:
That doesn't sound very welcoming. Why don't we just promote more long-term contributors to maintainer status?

Reply:
This statement is the reality that we have to face.  While we do have enthusiastic newer reviewers like @ltem, @MustaphaTR and @Smittytron helping out where they can, there are many PRs that they do not feel qualified to review.  Most non-trivial PRs sit for multiple weeks waiting for review, and stating this up front shows that this is not a personal snub against the contributor.